test.js

```diff
@@ -0,0 +1,28 @@
+const codeImport = require('./index.js');
```

Comment:
Prefer to just import from `./`, so that it could load from `package.json` directly for the sake of e2e testing.

```suggestion
const codeImport = require('./');
```